java/src/jmri/jmrit/display/layoutEditor/LayoutEditorComponent.java

```diff
@@ -633,6 +628,14 @@ private void highLightSelection(Graphics2D g) {
         g.setStroke(stroke);
     }
 
+    private void drawLayoutTrackText(Graphics2D g) {
```

Comment:
This (a) draws all tracks, not just one, and (b) has the same name as `LayoutTrackView#drawLayoutTrackText` which draws just one.  Could you please rename this one to i.e. `drawLayoutTracksText` to make the separation a little clearer? Thanks

Reply:
Good catch! Thanks… (done!)